### PR TITLE
Replace `"null"`s & Initalization checks

### PR DESCRIPTION
**Pull request by MrBartusek**
This PR replaces all `"null"` strings with actually `null!` properties. I've also added a `initialized` property and moved all initialization checks to the client.

### CODE REVIEW

**Review by TheDrone7 — CHANGES_REQUESTED** [review]
Everything else looks fine but please change the `null` to `undefined`. We want to avoid using `null` type at all costs.

**Comment by MrBartusek** [comment]
> We want to avoid using `null` type at all costs.

I'm curious, why is that?

**Comment by TheDrone7** [comment]
> > We want to avoid using `null` type at all costs.
> 
> I'm curious, why is that?

It's best to use either `null` or `undefined` consistently in a project. Using the two, together, only adds to confusion.
I just decided to go with `undefined` for shieldbow instead of `null`.

File: src/structures/Participant.ts [hunk 1]
```diff
@@ -576,7 +576,7 @@ export class Participant {
     this.dragonKills = data.dragonKills;
 
     const summonerData: SummonerData = {
-      accountId: 'null',
+      accountId: null!,
```
Comment (TheDrone7) [inline src/structures/Participant.ts:579]: It's still `null` here.

**Review by TheDrone7 — CHANGES_REQUESTED** [review]

**Review by TheDrone7 — APPROVED** [review]
Looks good!

**Pull request merged**